recipes/canopy/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version="0.25" %}
+{% set name="canopy" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hildebra/canopy2/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 6ea391021dc81ff1981bb69b0ec27a56a17bd23e01c25a27b86808366d62061f
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x.x") }}
+    
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - zlib
+  run:
+    - zlib
+
+test:
+  commands:
+    - ./cc.bin  --version | grep {{ version }}
```

Comment:
Since you use `mv` at https://github.com/bioconda/bioconda-recipes/pull/47245/files#diff-4581d3799114d6587ba2ddb4f6fb6037516f825ce9041f58ddaabca414ca11d9R6 I think the binary won't be in the current folder anymore.
If I am correct then try without `./`, i.e. it will look in the $PATH and should be found in $PREFIX/bin.

Reply:
Thanks @martin-g !